tsconfig.json

```diff
@@ -9,8 +9,7 @@
     "declaration": true,
     "jsx": "react",
     "rootDir": "src",
-    "outDir": "lib",
-    "incremental": true
```

Comment:
This package is too small to benefit from incremental builds. But this option confused my artifacts, I got no changes in the bundles after manipulating build configurations, which is weird. 

Easier to remove this